src/tools/execlog/src/main/java/com/google/devtools/build/execlog/ParserOptions.java

```diff
@@ -60,4 +60,13 @@ public class ParserOptions extends OptionsBase {
       effectTags = {OptionEffectTag.UNKNOWN},
       help = "If set, only output the executions that used the given runner.")
   public String restrictToRunner;
+
+  @Option(
+          name = "shard_sorted",
```

Comment:
What was the original order? Can we just sort them by default and not have this option at all?

Reply:
Hi meteorcloudy, thanks for the feedback! The original order was to reorder `/tmp/exec2.log` to match actions detected in `/tmp/exec1.log`. This causes the shard indices to be rearranged in an unorganized order in the output log. Therefore, we added a new option so that the user could choose whether to use the original rearranging approach or to sort by shards.

If this new sorting method is satisfactory and you wish to make it as default, we are happy to remove the newly added option while keeping the sorting code.

Reply:
I think having only the sorted result should be fine.

Reply:
Ok, we deleted the option and now it sorts by shards by default.